python/GafferUI/GLWidget.py

```diff
@@ -249,6 +246,90 @@ def keyPressEvent( self, event ) :
 
 		QtWidgets.QFrame.keyPressEvent( self, event )
 
+	@classmethod
+	def __createGLWidget( cls, _format ) :
+
+		# try to make a host specific widget if necessary.
+		result = cls.__createMayaQGLWidget( _format )
+		if result is not None :
+			return result
+
+		result = cls.__createHoudiniQGLWidget( _format )
+		if result is not None :
+			return result
+
+		glWidget = QtWidgets.QOpenGLWidget()
+		# Avoid `QOpenGLFramebufferObject: Framebuffer incomplete attachment`
+		# errors caused by Qt trying to make a framebuffer with zero size.
+		glWidget.setMinimumSize( 1, 1 )
+		glWidget.setFormat( _format )
+
+		return glWidget
+
+	@classmethod
+	def __createQGLFormat( cls, _format ):
+		"""
+		Create a QGLFormat based on the configuration of
+		QSufraceFormat where possible.
+		"""
+		qGLFormat = QtOpenGL.QGLFormat()
+		# note that here we are not using _format.hasAlpha()
+		# because it always return False ( probably uninitalized ).
```

Comment:
My tests suggest that `hasAlpha()` does return True if `QSurfaceFormat.setAlphaBufferSize()` has been called. It just seems that we're not using `BufferOptions.Alpha` anywhere so it's hasn't been called. So perhaps this comment is inaccurate, or you're seeing something different in your Qt version, or `setRgba()` is needed for some other reason?